gtsam/base/VectorSpace.h

```diff
@@ -51,6 +51,8 @@ struct VectorSpaceImpl {
   /// @name Lie Group
   /// @{
 
+  typedef Eigen::Matrix<double, N, 1> LieAlgebra;
```

Comment:
`using` instead of `typedef` to be consistent?

Reply:
Good catch. I am not gonna launch another entire CI for this one change but maybe it can be cleaned up for all of group/manifold/lie/VectorSpace in a “using” PR.

Reply:
You can merge this and I can update it in my new PR.